### PR TITLE
Genesis

### PR DESCRIPTION
**Pull request by mattruggio**
This PR houses the initial transformer extraction from our application codebase.  The transformers have been re-organized to provide semantic grouping, other than that, it is just an extraction.

### CODE REVIEW

File: lib/realize/collection/sort/direction.rb [hunk 1]
```diff
@@ -0,0 +1,36 @@
+# frozen_string_literal: true
+
+require_relative 'sort/direction'
+
+module Realize
+  class Collection
+    # Transformer to take an array of oibjects and sort by the given key
```
Comment (gerryster) [inline lib/realize/collection/sort/direction.rb:7]: oibjects -> objects

Comment (gerryster) [reply lib/realize/collection/sort/direction.rb:7]: It looks like this spelling error occurs in several different class comments.

File: lib/realize/collection/sort/direction.rb [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    # and by the given sort direction. Defaulting to ascending.
+    class Sort
```
Comment (gerryster) [inline lib/realize/collection/sort/direction.rb:9]: This looks like the same as is defined in lib/realize/collection/sort.rb .

Comment (mattruggio) [reply lib/realize/collection/sort/direction.rb:9]: Yes, this was in error.

**Pull request merged**